sql/core/src/main/scala/org/apache/spark/sql/execution/GeneratedAggregate.scala

```diff
@@ -41,13 +42,15 @@ case class AggregateEvaluation(
  * @param groupingExpressions expressions that are evaluated to determine grouping.
  * @param aggregateExpressions expressions that are computed for each group.
  * @param child the input data source.
+ * @param unsafeEnabled whether to allow Unsafe-based aggregation buffers to be used.
  */
 @DeveloperApi
 case class GeneratedAggregate(
     partial: Boolean,
     groupingExpressions: Seq[Expression],
     aggregateExpressions: Seq[NamedExpression],
-    child: SparkPlan)
+    child: SparkPlan,
+    unsafeEnabled: Boolean)
```

Comment:
nit: typically place children last (although we aren't great about following this convention)...